docs/squash-api/index.rst

```diff
@@ -0,0 +1,86 @@
+##########
+squash-api
+##########
+
+.. list-table::
+   :widths: 10,40
+
+   * - Edit on GitHub
+     - `/services/squash-api <https://github.com/lsst-sqre/lsp-deploy/tree/master/services/squash-api>`__
+   * - Type
+     - Helm_
+   * - Namespace
+     - ``squash-api``
+
+.. rubric:: Overview
+
+The ``squash-api`` app deploys a REST API for storing and visualizing Science Pipelines metrics. You can learn more about SQuaSH in SQR-009_.
```

Comment:
Science Pipelines metrics --> measurements of Science Pipelines metrics

Reply:
Here I think "Science Pipeline metrics" is correct because we also store the metric definitions and specifications in the SQuaSH API.